index.html

```diff
@@ -1832,55 +1832,125 @@ <h2>Proof</h2>
+A <a>DID document</a> MUST be a single JSON object conforming to [[!RFC8259]]. All
+top-level properties of the <a>DID document</a> MUST be represented by using the
+property name as the name of the member of the JSON object. The values of
+properties of the data model described in Section <a href="#data-model"></a>, 
+including all extensions, MUST be encoded in JSON[[RFC8259]] by mapping property
 values to JSON types as follows:
         </p>
 
         <ul>
           <li>
-Numeric values representable as IEEE754 SHOULD be represented as a Number type.
+Numeric values representable as IEEE754 MUST be represented as a Number type.
           </li>
           <li>
-Boolean values SHOULD be represented as a Boolean type.
+Boolean values MUST be represented as a Boolean type.
           </li>
           <li>
-Sequence value SHOULD be represented as an Array type.
+Sequence value MUST be represented as an Array type.
           </li>
           <li>
-Unordered sets of values SHOULD be represented as an Array type.
+Unordered sets of values MUST be represented as an Array type.
           </li>
           <li>
-Sets of properties SHOULD be represented as an Object type.
+Sets of properties MUST be represented as an Object type.
           </li>
           <li>
-Empty values SHOULD be represented as a null value.
+Empty values MUST be represented as a null value.
           </li>
           <li>
 Other values MUST be represented as a String type.
           </li>
         </ul>
+
+        <p>
+All properties of the <a>DID document</a> MUST be included in the object.
+        </p>
+
+        <p>
+The property name <code>@context</code> MUST NOT be used as this is reserved
+for JSON-LD processors. 
+        </p>
+        
+</section>
+
+<section>
+  <h3>Consumption</h3>
+  
+    <p>
+The top-level element MUST be a JSON object. Any other data type at the top 
+level is an error and MUST be rejected. The top-level JSON object is taken 
+as the <a>DID document</a>. All members of this object are properties of the
+<a>DID document</a>. The object member name is the property name, and the
+member value is interpreted as follows:
+    </p>
+
+        <ul>
+          <li>
+Number types MUST interpreted as numeric values representable as IEEE754.
+          </li>
+          <li>
+Boolean types MUST be interpreted as a Boolean value.
+          </li>
+          <li>
+An Array type MUST be interpreted as a Sequence or Unordered set, depending on the
+definition of the property for this value.
+          </li>
+          <li>
+An Object type MUST be interpreted as a sets of properties.
+          </li>
+          <li>
+A null type MUST be interpreted as an Empty value.
+          </li>
+          <li>
+String types MUST be interpreted as Strings, which may be further parsed
+depending on the definition of the property for this value.
+          </li>
+        </ul>
+
+  <p>
+The value of the <code>@context</code> field MUST be ignored.
```

Comment:
If they try to "upconvert to JSONLD" then they are doing it wrong. 

What we're saying is that a JSON-formatted DID Document cannot rely on the context field. If you want to rely on the context field, then you need to create a JSONLD-formatted DID document and follow those rules.

Reply:
What you're reading from this statement is exactly correct: this is about JSON consumers, not JSONLD consumers.

To be more specific, the following requirement earlier in the section is intended to prevent exactly this kind of cross-format upconversion:

> translation between any two representations
MUST be done by parsing the source format into a <a>DID document</a> model
(described in Sections <a href="#data-model"></a> and <a href="#did-documents"></a>)
and then serializing the <a>DID document</a> model into the target representation.

So in other words, if I have a signal that the document is JSON but it has a context field, then I am not allowed to process it as JSONLD and I need to ignore the context field, which should not have been included in the first place because it's forbidden by the JSON production rules, above.

If I've got a document that's JSONLD, then I need to read the context field, as in the next section. And if I'm producing JSONLD, then I need to include the context field. 

This is why we need to have document metadata to convey representation information between producers and consumers. I did not want to add that to the document with this pull request as that is a separate issue and I strove to make this pull request as small as actionably possible.